source/designers-guide/snippets/index.md

```diff
@@ -224,6 +224,10 @@ A few things to keep in mind when using this approach:
 
 If your plugin/theme uses snippets, they should be placed inside the corresponding directory in your plugin/theme. If that is done correctly, when the plugin is installed in another Shopware installation, those snippets will be automatically imported from the .ini file into the database. This minimizes the number of file reads in production environments, maximizing performance.
 
+<div class="alert alert-warning">
```

Comment:
Please do not use a warning here. It is nothing "dangerous" here :wink: 
Just add an example for the snippet path in a plugin and the path in a theme.

Reply:
Hi mitelg,

I just wanted to keep in line with the document style, hence the use of warnings.

I changed it to a simple code style example and added an example for a theme as well.

Cheers